src/SFA.DAS.Employer.PR.Web/Controllers/ChangePermissionsController.cs

```diff
@@ -0,0 +1,84 @@
+﻿using FluentValidation;
+using FluentValidation.AspNetCore;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Employer.PR.Domain.Interfaces;
+using SFA.DAS.Employer.PR.Domain.OuterApi.Permissions;
+using SFA.DAS.Employer.PR.Web.Authentication;
+using SFA.DAS.Employer.PR.Web.Constants;
+using SFA.DAS.Employer.PR.Web.Infrastructure;
+using SFA.DAS.Employer.PR.Web.Models;
+using SFA.DAS.Employer.PR.Web.Services;
+using SFA.DAS.Encoding;
+using System.Security.Claims;
+
+namespace SFA.DAS.Employer.PR.Web.Controllers;
+
+[Authorize(Policy = nameof(PolicyNames.HasEmployerOwnerAccount))]
+[Route("accounts/{employerAccountId}/providers/changePermissions", Name = RouteNames.ChangePermissions)]
```

Comment:
The route should be `accounts/{employeraccountId}/providers/{ukprn}/changepermissions` so you can read ukprn from the route.

Reply:
good suggestion, done